Add some new issue templates to get us started (#16)
upbound/upbound: .github/ISSUE_TEMPLATE/bug_report.md

name: Bug Report

> description: Identify bugs in of our Upbound's products.

Reviewer: ```suggestion
description: Identified a bug in one Upbound's products?
```

labels: bug,needs:triage

> title: "<component name>: <bug overview>"

Reviewer: ```suggestion
title: "<product name>: <bug overview>"
```

> body:

Reviewer: suggestion(non-blocking): Should we have a dropdown field with a list of Upbound products to report on? If so we can also remove <product name> from the issue title.
